subsys/net/ip/Kconfig

```diff
@@ -752,6 +752,12 @@ config NET_BUF_DATA_POOL_SIZE
 	  This value tell what is the size of the memory pool where each
 	  network buffer is allocated from.
 
+config NET_PKT_BUF_USER_DATA_SIZE
+	int "Size of user_data available in rx and tx network buffers"
+	default 4
```

Comment:
Maybe also a `range 4 <some sensible max>` to enforce a minimum of 4?

Reply:
Added `range 4 16`. I think the max should be at least 8, so it fits a 64 bit pointer.